### ISSUE TITLE
Could not find argument id for type

### ISSUE THREAD

**Issue by linrium** [graphql-nexus/nexus-plugin-prisma#80]
![image](https://user-images.githubusercontent.com/14315537/52172619-f846fe00-27a5-11e9-8f08-d728b4ee2d54.png)
When I use this query
```
mutation {
  updateStory(where: {
    id: "cjrodoikb001z0860g7wzuk21"
  }, data: {
    content: "asd"
  }) {
    id
  }
}
```
I got this error message
```
{
  "data": {
    "updateStory": null
  },
  "errors": [
    {
      "message": "Could not find argument id for type Story",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "updateStory"
      ]
    }
  ]
}
```

**Comment by devdanio** [comment]
@lynkxyz  Im having the same issue, did you find a fix yet?

**Comment by linrium** [comment]
@danramosd No, I didn't find any solution.

**Comment by Weakky** [comment]
Hey there,

This is now fixed by 35d96740bdfd4c6e58a29236294b59c8560a93d8.
I'll close the issue once it's published 🙌

**Comment by Weakky** [comment]
Published under `nexus-prisma@0.1.3`, closing now
